Formula/hunspell.rb

```diff
@@ -1,10 +1,9 @@
 class Hunspell < Formula
   desc "Spell checker and morphological analyzer"
   homepage "https://hunspell.github.io"
-  url "https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz"
-  sha256 "bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a"
-  license "GPL-2.0"
-  revision 2
+  url "https://github.com/hunspell/hunspell/archive/v1.7.1.tar.gz"
+  sha256 "6e3557624c608b3e6525b8bd277706db4f5a857c28fdb3cfa8d0d2b67776da8a"
```

Comment:
Looks like upstream now has an official tarball. May want to try that one and use the included `configure` rather than requiring `autoconf, automake, libtool` dependencies + `autoreconf ...`
```suggestion
  url "https://github.com/hunspell/hunspell/releases/download/v1.7.1/hunspell-1.7.1.tar.gz"
  sha256 "b2d9c5369c2cc7f321cb5983fda2dbf007dce3d9e17519746840a6f0c4bf7444"
```

At least many Linux distros like [Arch](https://github.com/archlinux/svntogit-packages/blob/packages/hunspell/trunk/PKGBUILD#L13), [Fedora](https://src.fedoraproject.org/rpms/hunspell/blob/rawhide/f/hunspell.spec#_7), and [Gentoo](https://gitweb.gentoo.org/repo/gentoo.git/tree/app-text/hunspell/hunspell-1.7.1.ebuild#n12) are using that one rather than GitHub's autogenerated copy.